server/controllers/chartsController/index.js

```diff
@@ -45,7 +46,8 @@ export const graphDataController = async (dataDb, userAccess, chart_id) => {
     chart.fieldLabelValueMap.forEach((fieldLabelValueMap) => {
       const { color, label, value, targetValues } = fieldLabelValueMap
       const targetValue = targetValues[study]
-      const dataKey = `${study}-${label}-${targetValue}`
+      const siteName = SITE_NAMES[study] || study
```

Comment:
What does it mean if there is a study/site that is not in `SITE_NAMES`?

Reply:
good point, if the site is missing from SITE_NAMES we default to the abbreviated name

Reply:
Right, that sounds like that would only happen with an unexpected value, should it be an error case of some kind or is it expected/okay that not all sites will have an entry in that lookup?

Reply:
Thanks for the help on this pr. Could use your help me understanding what you mean by unexpected value?

Reply:
Sure, I'm wondering when we would hit that default -- I know we'd hit it if `study` does not exist in `SITE_NAMES` but when would that be the case and what are the implications of that, does it mean whatever is creating the value for `study` has a bug, does it mean we accidentally deleted an entry in `SITE_NAMES`, etc. I don't think there's anything wrong with using the default here, but just wondered if the "hitting of the default" would indicate an issue in the app or its data that we'd want to be made aware of.

Reply:
It's a good point, but I think the default is going to work best in this case. I think it's reasonable that they would add a new site and we would not have a mapping for it, in which case we will want to show the abbreviation instead of throwing an error.

Reply:
> they would add a new site and we would not have a mapping for it

That makes sense!